### DISCUSSION TITLE
Intégration de l'écran et du tchat dans la page ATRPG coté joueur

### DISCUSSION THREAD

**Discussion by vouiv-re** [fibreville/minotaure_php#91] [Boite à idées]
Est-ce qu'on ne pourrait pas imaginer intégrer la vidéo et le tchat, sur page ATRPG des joueurs ? Histoire d'avoir tout en un, pour faciliter la navigation des joueurs qui n'ont qu'un écran ? Je ne sais pas si c'est faisable techniquement, mais je ferais des recherches si ça intéresse du monde.

**Comment by ElethornGM** [comment]
Ou alors, peut-être plus simple, de pouvoir réduire la feuille de perso en une petite fenêtre et de pouvoir l'épinglée. L'un ou l'autre peut être une bonne idée en effet, car certains joueurs n'aiment pas trop / ont du mal à s'habituer à naviguer entre deux pages.
